### ISSUE TITLE
control surface dynamics are missing a bunch of terms

### ISSUE THREAD

**Issue by RussTedrake** [RobotLocomotion/drake#522]
i've pushed a controlsurface branch on RussTedrake/drake to follow up on the pull request i merged this morning.  my goal was to use the controlsurface to (finally) resolve all of the numerical issues with the glider urdf model.  i ended up finding some problems with the control surface implementation.  

First - i did verify that if you use the current glider urdf, but set the inertia of the tail to zero (and change RBM to avoid welding the massless dof), then the glider dynamics do match to tolerance:  https://github.com/RussTedrake/drake/commit/6945b41594f1df9aaed2c2d4814eb9bb241e6eb9
but it requires using pinv instead of inv in the main manipulator.sodynamics method, and (as expected) results in inf/nan qdd on the control surface.  

A better solution would be to rip out the tail and just add it as a massless control surface.  The glider model assumes a control surface under velocity control, but this would be very easy to add the the current control surface class.  (I've added an inline comment about that).  In fact, it feels very strange to not have that.

One of the other things I did to make this all work was to replace the spline interpolated lift and drag coefficients for function_handle coefficients for the case of the flat plate.  it should absolutely be that way (numerically better, but also reserves the possibility of non-local symbolic control design / verification)

As I went to propagate the updates to the control surface class, I found a bunch of missing terms.  Most notably, the state-dependent contribution of the control surface is missing.  and you are doing finite differences for the linearization when both the function_handle and the spline interpolants have trivial analytical derivatives.  

Long story short, I've got a lot of updates I want to discuss typed in to my controlsurface branch.  I'm hoping to discuss then push them to completion together.
